### DISCUSSION TITLE
NA

### DISCUSSION THREAD

**Discussion by philip-ndikum** [Nixtla/neuralforecast#701] [General]

**Comment by kdgutier** [comment]
Hey @anonymouspapi,

Here is a description of cross-validation:
https://nixtla.github.io/statsforecast/docs/tutorials/crossvalidation.html

**Comment by tibeler** [comment]
i did not understand it either! what is the difference betwteen nf.fit and nf.cross_validation? it is important ,and the docs are too difficult to understand
